share/locale/en.json

```diff
@@ -119,35 +119,35 @@
     "settings_error_socket_file": "Can't connect to Tor controller using socket file {}.",
     "settings_error_auth": "Connected to {}:{}, but can't authenticate. Maybe this isn't a Tor controller?",
     "settings_error_missing_password": "Connected to Tor controller, but it requires a password to authenticate.",
-    "settings_error_unreadable_cookie_file": "Connected to Tor controller, but can't authenticate because your password may be wrong, and your user doesn't have permission to read the cookie file.",
-    "settings_error_bundled_tor_not_supported": "Bundled Tor is not supported when not using developer mode in Windows or macOS.",
-    "settings_error_bundled_tor_timeout": "Connecting to Tor is taking too long. Maybe your computer is offline, or your clock isn't accurate.",
+    "settings_error_unreadable_cookie_file": "Connected to Tor controller, but can't authenticate because your password may be wrong, and your user lacks permission to read the cookie file.",
+    "settings_error_bundled_tor_not_supported": "Use of the Tor version bundled with OnionShare is not supported when not using developer mode on Windows or macOS.",
```

Comment:
`Use of the Tor version bundled with OnionShare`

`Bundled Tor is not supported when not using developer mode` sounds better.

Reply:
@Baccount It begs the question of "Bundled Tor" is.

Reply:
Bundled Tor is the Pre compelled version of tor that is inside the .app

Reply:
This may not be not all that clear for the user, whereas if it says it in cleartext, no confusion will arise.
That is the intent anyhow.

Reply:
I'm fine with `Use of the Tor version bundled with OnionShare`, because that's the same language used in ` "gui_settings_connection_type_bundled_option": "Use the Tor version that is bundled with OnionShare"`. Also, this string is only ever displayed to people who run `./dev_scripts/onionshare-gui`, and not to people who run OnionShare from the installer, and I think developers are likely to understand it either way.

But, I realized there's a typo. Bundled Tor isn't supported when *using* developer mode, not when *not using* developer mode. So the string should be this instead: `Use of the Tor version bundled with OnionShare is not supported when using developer mode on Windows or macOS.`